### PR TITLE
docs: Add Docker to Jira access request plugin

### PR DESCRIPTION
**Pull request by lsgunn-teleport**

### CODE REVIEW

**Comment by ptgott** [comment]
It might make sense to hold off on this change until we merge my edits to make the Jira guide accurate and possible to follow: https://github.com/gravitational/teleport/pull/29827

**Review by zmb3 — APPROVED** [review]

**Review by ptgott — COMMENTED** [review]
In case my earlier comment got lost in the conversation, the Jira guide is still out of date. I have a PR here to fix the inaccuracies and make it possible to follow the guide all the way through. Merging this PR will create merge conflicts with that one: https://github.com/gravitational/teleport/pull/29827

**Comment by lsgunn-teleport** [comment]
How about I just close this PR then? I found the instructions impossible to follow as text and didn't set up an environment to fix them.

**Pull request closed without merging**
